package.json

```diff
@@ -134,10 +134,11 @@
     "color-hash": "^1.0.3",
     "copy-to-clipboard": "^3.0.8",
     "grommet": "^2.14.0",
-    "json-e": "^4.1.0",
     "hast-util-sanitize": "^3.0.0",
+    "json-e": "^4.1.0",
     "lodash": "^4.17.11",
     "mermaid": "8.4.0",
+    "moment": "^2.27.0",
```

Comment:
Let's not use `moment` because it is pretty massive (especially if you don't remove the locales), let's use `https://date-fns.org/` instead

Reply:
Yeah - I'd prefer `date-fns` too. I can switch to that in this PR but it will probably mean I will end up migrating all our Jellyfish repos from `moment` to `date-fns` too as it doesn't make much sense to use one library in Rendition and another in Jellyfish!